operator/src/test/java/org/bf2/operator/operands/KafkaInstanceTest.java

```diff
@@ -90,6 +93,16 @@ void statusUnknownWithReason() {
         assertEquals("I don't know", readiness.getMessage());
     }
 
+    @Test
+    void statusUnknownWhenPaused() {
+        ManagedKafka pausedInstance = ManagedKafkaUtils.dummyManagedKafka("x");
+        pausedInstance.getMetadata().setAnnotations(Map.of(ManagedKafkaKeys.Annotations.PAUSE_RECONCILIATION, "true"));
+        OperandReadiness readiness = kafkaInstance.getReadiness(pausedInstance);
+        assertEquals(Status.Unknown, readiness.getStatus());
+        assertEquals(Reason.Paused, readiness.getReason());
+        assertEquals("Reconciliation paused via annotation", readiness.getMessage());
```

Comment:
do you think an extension to the test to remove annotation and flow to normal status is useful?

Reply:
Updated - also updated the other test to check the metric